### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by maxbrito500**
Adding detailed instructions for pom.xml

### CODE REVIEW

**Comment by maxbrito500** [comment]
This patch is related to the conversation at https://github.com/tcheeric/nostr-java/issues/176

File: README.md [hunk 1]
```diff
@@ -8,13 +8,52 @@ Introducing the nostr-java library, a solution written in java for generating, s
...
 
 ## Usage
-To use the library in your project, add the following dependency to your pom.xml file:
+To use the library in your project, add the following dependency to your pom.xml file.
+
+Define the value for the most recent version available:
+```xml
+    <properties>
+        <nostr-java.version>0.6.1-SNAPSHOT</nostr-java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```
Comment (avlo) [inline README.md:17]: property:
  `<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>` 
is already defined in [nostr-java's pom.xml](https://github.com/tcheeric/nostr-java/blob/71c1c49b6480778cc9a445df691a55adc65aabe2/pom.xml#L33).

your project might have additional explicit dependency for it above and beyond what's required for nostr-java, but that is exclusive to your project and unnecessary here as part of general instruction.

File: README.md [hunk 2]
```diff
@@ -8,13 +8,52 @@ Introducing the nostr-java library, a solution written in java for generating, s
...
+    </properties>
+```
+
+Add Jitpack to the repositories:
```
Comment (avlo) [inline README.md:21]: Obtaining nostr-java library jars.  Two options:

Option 1: For pre-compiled binaries, add Jitpack to pom.xml repositories:
```xml
    <repositories>
        <repository>
            <id>jitpack.io</id>
            <url>https://jitpack.io</url>
        </repository>
    </repositories>
````
Option 2: To build and install your own nostr-java dependency libraries:
```
$ cd <your_git_home_dir>
$ git clone git@github.com:tcheeric/nostr-java.git
$ cd nostr-java
$ git checkout <relevant_branch>
$ mvn clean install
```

**Comment by tcheeric** [comment]
Thank you
-- 
 Sent with Tuta; enjoy secure & ad-free emails: 
 https://tuta.com


26 Aug 2024, 22:41 by ***@***.***:

**Review by tcheeric — APPROVED** [review]

**Comment by maxbrito500** [comment]
This is still on the development branch, why not on the main one?

Right now it is kept invisible to new users since they just see the main branch by default with the old instructions.

**Comment by tcheeric** [comment]
It will be merged to main in the weekend. 

-- 
 Sent with Tuta; enjoy secure & ad-free emails: 
 https://tuta.com


27 Sept 2024, 17:29 by ***@***.***:

**Comment by maxbrito500** [comment]
Thank you, just puzzled why it didn't went directly. Please do continue the good work, this is without doubt the best java library for nostr.

**Comment by tcheeric** [comment]
It's just the process that I'm following
PRs ----> develop ----> main🤙🙏-- 
 Sent with Tuta; enjoy secure & ad-free emails: 
 https://tuta.com


27 Sept 2024, 19:43 by ***@***.***:

**Pull request merged**